language/tools/test_generation/src/abstract_state.rs

```diff
@@ -99,6 +99,9 @@ pub struct AbstractState {
 
     /// The module state
     pub module: CompiledModule,
+
+    /// This flag is set when applying an instruction should result in an error
```

Comment:
I don't understand this comment. Please try to clarify it.

Reply:
Fix in upcoming PR.